SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  ViewListingView.swift
+//  SH4RE
+//
+//  Created by Hannah Brooks on 2022-11-24.
+//
+
+import SwiftUI
+import Firebase
+import FirebaseStorage
+
+//Stuff to know for database
+//ViewListing is passed a listing struct define in ListingViewModel
+//We map the fields from the listing struct to class variable in .OnAppear
+//View listing is passed and image which is set as the classes image variable being displayed
+//if you need to create more variables create filler class variable and we will connect to database in later commit
+
+struct ViewListingView: View {
+    
+    //parameters passed in from search nav link
+    var listing: Listing
+    var image: UIImage
+    
+    let screenSize: CGRect = UIScreen.main.bounds
+    var numberOfStars = 4
+    var hasHalfStar = true
+    var numberOfReviews = 3
+    var numberOfImages = 3 // should become images.length or something
+    @State var description:String = ""
+    @State var title:String = ""
+    @State var price:String = ""
+    
+    var body: some View {
+        
+        ZStack {
+            Color("BackgroundGrey").ignoresSafeArea()
+            VStack(alignment: .leading) {
+                
+                GeometryReader { geometry in
+                    ImageCarouselView(numberOfImages: self.numberOfImages) {
+                        ForEach([self.image, UIImage(named: "ProfilePhotoPlaceholder")!, UIImage(named: "ProfilePhotoPlaceholder")!], id:\.self) { image in
```

Comment:
do we need placeholders? (not sure about the support for uploading mult images yet, just wondering)

Reply:
i just have them there bc i need bryans changes to go forward. so this will be replaced with `images` or whatnot